user_interactions/user_interaction_txt_text.py

```diff
@@ -0,0 +1,69 @@
+import os
+from datetime import datetime
+
+from user_interactions.base_user_interaction import (BaseUserInteraction,
+                                                     MessageScope)
```

Comment:
Not sure if these parentheses needed. We don't have them in same situations (where 2 or more instances were imported) in other code parts

Reply:
> Not sure if these parentheses needed. We don't have them in same situations (where 2 or more instances were imported) in other code parts

These parentheses were created automatically because of the string length.